Big-Spender/readme.md

```diff
@@ -67,42 +68,55 @@ INSERT YOUR QUERY HERE
 
 **You:** Then here's the query for that:
 
-```sql
+````sql
 INSERT YOUR QUERY HERE
-```
+```   SELECT * FROM spends WHERE LOWER(description) LIKE '%fee%' OR LOWER(description) LIKE '%fees%'```
+````
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
-```sql
+````sql
 INSERT YOUR QUERY HERE
-```
+```SELECT * FROM spends WHERE expense_area_id =(SELECT expense_area_id FROM expense_areas WHERE expense_area = 'Better Hospital Food') ```
+````
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
-```sql
+````sql
 CREATE YOUR QUERY HERE
-```
+``` SELECT date, SUM(amount) As amount FROM spends GROUP BY date  ```
```

Comment:
This query is not entirely accurate since the date column includes other information than the month (e.g. month, year). You can extract the month field out of data using EXTRACT sql command and group only by the month.

https://www.postgresqltutorial.com/postgresql-date-functions/postgresql-extract/